pyproject.toml

```diff
@@ -41,13 +40,11 @@ Changelog = "https://pypdf.readthedocs.io/en/latest/meta/CHANGELOG.html"
 
 [project.optional-dependencies]
 full = [
-    "cryptography; python_version >= '3.7'",
-    "PyCryptodome; python_version == '3.6'",
+    "cryptography; python_version >= '3.8'",
     "Pillow>=8.0.0",
 ]
 crypto = [
-    "cryptography; python_version >= '3.7'",
-    "PyCryptodome; python_version == '3.6'",
+    "cryptography; python_version >= '3.8'",
```

Comment:
PyCryptoDome should be proposed as an `crypto_alternative`

Reply:
Is that change within the scope of this MR? I have no input on `cryptography` vs `PyCryptodome`

Reply:
Let's keep it on side for the moment.

Reply:
This is out of scope for now as it might need further discussions.